src/components/WelcomePage/Index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import uw211Logo from './images/unitedway211.png';
+
+function WelcomePage(props: { name: string }) {
```

Comment:
I would probably change the name of the prop to "id" instead of "name" to reflect what we'll end up doing